### PR TITLE
chore: fix styles

### PR DESCRIPTION
**Pull request by YuriCat**
few changes by PEP8

How about , at the end of lists?

### CODE REVIEW

File: handyrl/environments/tictactoe.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def play(self, action):
 
         # check winning condition
         if self.board[x, :].sum() == 3 * self.color \
-          or self.board[:, y].sum() == 3 * self.color \
-          or (x == y and np.diag(self.board, k=0).sum() == 3 * self.color) \
-          or (x == 2 - y and np.diag(self.board[::-1, :], k=0).sum() == 3 * self.color):
+            or self.board[:, y].sum() == 3 * self.color \
+            or (x == y and np.diag(self.board, k=0).sum() == 3 * self.color) \
+                or (x == 2 - y and np.diag(self.board[::-1, :], k=0).sum() == 3 * self.color):
```
Comment (ikki407) [inline handyrl/environments/tictactoe.py:60]: (It's a small thing!) Why does this indent has more 4 spaces from the previous line?

File: handyrl/util.py [hunk 2]
```diff
@@ -43,24 +43,24 @@ def rotate(x, max_depth=1024):
     if isinstance(x, (list, tuple)):
         if isinstance(x[0], (list, tuple)):
             return type(x[0])(
-                rotate(type(x)(xx[i] for xx in x), max_depth - 1) \
```
Comment (ikki407) [inline handyrl/util.py:46]: Yeah! It's redundant between brackets.

File: handyrl/environments/tictactoe.py (hunk 1 above)
Comment (YuriCat) [reply handyrl/environments/tictactoe.py:60]: PEP8 requires this indent.

**E125 continuation line with same indent as next logical line**

https://www.flake8rules.com/rules/E125.html

**Comment by ikki407** [comment]
It sounds good!

**Pull request merged**